src/examples/using-with-cmake/CMakeLists.txt

```diff
@@ -49,5 +49,6 @@ target_link_libraries(ascent_render_example ascent::ascent)
 # we need to make sure CUDA_RESOLVE_DEVICE_SYMBOLS is on for our target
 # (it propgates some way magically in 3.14, but not in 3.21)
 if(CMAKE_CUDA_COMPILER)
-    set_property(TARGET ascent_render_example PROPERTY CUDA_RESOLVE_DEVICE_SYMBOLS ON)
+	set_property(TARGET ascent_render_example PROPERTY CUDA_RESOLVE_DEVICE_SYMBOLS ON)
+	#set_property(TARGET ascent_render_example PROPERTY CUDA_ARCHITECTURES "70")
```

Comment:
I think this is the line causing the cuda error you were asking about.

Reply:
@mclarsen Yep! Something changed with Azure so we were running out of build space. Changing the build type from Release to MinSizeRel helped: https://github.com/Alpine-DAV/ascent/blob/ecc36b4e6d2987ae2b55f2334c3abca737f0163b/azure-pipelines.yml#L449